### PR TITLE
[MOBILE-1685] update titanium to track module version

### PR DESCRIPTION
**Pull request by crow**
### What do these changes do?
Adds module tracking to ios and android

### Why are these changes necessary?
To be able to see what version the the titanium module people are using (if any)

### How did you verify these changes?
Built and installed ios, built android in the IDE - but having some issues with the android build from the makefile. It appears the issue (need to accept licences) is localized to my machine (also happening on master). Accepting the licenses via the sdk manager and command line hasn't worked so far. Will fix and update.

### Anything else a reviewer should know?
Titanium will set the module version as a build config variable but I don't know how to access it from the module during runtime. I added a version tracking java class because the build config that's generated by titanium isn't available from the module, it also matches what we do in ios.

### CODE REVIEW

File: ios/Classes/TiAirshipModuleVersion.m [hunk 1]
```diff
@@ -0,0 +1,13 @@
+/* Copyright Airship and Contributors */
+
+#import "TiAirshipModuleVersion.h"
+
+static NSString *const tiAirshipModuleVersionString = @"6.0.1";
```
Comment (marc-scig) [inline ios/Classes/TiAirshipModuleVersion.m:5]: we'll need to bump this to the major version

File: android/manifest [hunk 2]
```diff
@@ -2,7 +2,7 @@
 # this is your module manifest and used by Titanium
 # during compilation, packaging, distribution, etc.
 #
-version: 6.0.0
+version: 6.0.1
```
Comment (marc-scig) [inline android/manifest]: we'll need to bump this to the major version

File: scripts/update_version.sh [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    echo "No version number supplied"
+    exit
+fi
+
+# iOS and Android Manifests
+sed -i '' "s/\(^version:\).*/\1 $VERSION/g" $ROOT_PATH/ios/manifest
+sed -i '' "s/\(^version:\).*/\1 $VERSION/g" $ROOT_PATH/android/manifest
+
+# Android - TiAirshipModuleVersion.java
+sed -i '' "s/\(tiAirshipModuleVersion *= *\)\".*\"/\1\"$VERSION\"/g" $ROOT_PATH/android/src/ti/airship/TiAirshipModuleVersion.java
```
Comment (marc-scig) [inline scripts/update_version.sh]: is there already a TiAirshipModuleVersion.java? or was it an uncommitted new file?

File: android/manifest (hunk 2 above)
Comment (crow) [reply android/manifest]: there's no new functionality in the module itself and there aren't any api changes

Comment (marc-scig) [reply android/manifest]: you're right, nm

Comment (marc-scig) [inline android/manifest]: 7.0.0

File: android/src/ti/airship/TiAirshipModuleVersion.java [hunk 4]
```diff
@@ -0,0 +1,6 @@
+package ti.airship;
+
+public class TiAirshipModuleVersion {
+    static final String TI_AIRSHIP_MODULE_VERSION = "6.0.1";
```
Comment (marc-scig) [inline android/src/ti/airship/TiAirshipModuleVersion.java]: 7.0.0

File: ios/Classes/TiAirshipModuleVersion.m (hunk 1 above)
Comment (marc-scig) [inline ios/Classes/TiAirshipModuleVersion.m]: 7.0.0

File: ios/manifest [hunk 5]
```diff
@@ -2,7 +2,7 @@
 # this is your module manifest and used by Titanium
 # during compilation, packaging, distribution, etc.
 #
-version: 6.0.0
+version: 6.0.1
```
Comment (marc-scig) [inline ios/manifest]: 7.0.0

**Review by marc-scig — CHANGES_REQUESTED** [review]

File: DEV_README.md [hunk 6]
```diff
@@ -9,6 +9,9 @@ Module uses a makefile for build and install commands:
 - `install-ios`:  Builds and unpacks the iOS zip into the titanium module directory
 - `clean`: Cleans any build folders
 
+## Updating Module Version
+Run "./scripts/update_version <VERISON>" from root directory
```
Comment (BrianBatchelder) [inline DEV_README.md]:
```suggestion
Run "./scripts/update_version <VERSION>" from root directory
```

File: android/src/ti/airship/TiAirship.java [hunk 7]
```diff
@@ -9,13 +9,15 @@
...
 import com.urbanairship.UAirship;
+import com.urbanairship.analytics.Analytics;
 import com.urbanairship.app.GlobalActivityMonitor;
 import com.urbanairship.app.SimpleApplicationListener;
 import com.urbanairship.channel.AirshipChannelListener;
 import com.urbanairship.push.NotificationActionButtonInfo;
 import com.urbanairship.push.NotificationInfo;
 import com.urbanairship.push.NotificationListener;
 
+
```
Comment (BrianBatchelder) [inline android/src/ti/airship/TiAirship.java]:
```suggestion
```

**Review by BrianBatchelder — APPROVED** [review]
A couple of minor things. Pre-approved.
👍

**Review by BrianBatchelder — APPROVED** [review]
FWIW, 👍

**Review by rlepinski — APPROVED** [review]

**Pull request merged**